seed/energy/meter_data_processor/green_button_driver.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import logging
+from seed.energy.meter_data_processor import green_button_parser as parser
+import xml.etree.ElementTree as ET
+
+_log = logging.getLogger(__name__)
+
+def request_green_button_by_url(url):
+    #TODO SSL Verification is disabled
```

Comment:
Currently the test GreenButton account doesn't have SSL enabled. It will be future work to implement SSL support, once there is a testing account with SSL enabled...